src/feedback/admin.py

```diff
@@ -2,10 +2,44 @@
 
 from django.contrib import admin
 
-from feedback.models import Person, Veranstaltung, Semester, Einstellung, Mailvorlage, Kommentar, Tutor, BarcodeScanner, \
-    BarcodeScannEvent
+from feedback.models import Person, Veranstaltung, Semester, Einstellung, \
+    Mailvorlage, Kommentar, Tutor, BarcodeScanner, BarcodeScannEvent, BarcodeAllowedState
+from feedback.models.base import Log
 
-from django import forms
+
+def status_angelegt(modeladmin, request, queryset):
+    queryset.update(status=100)
```

Comment:
please use Veranstaltung.STATUS_ANGELEGT instead of 100 same for the other functions